modules/stonks/utils/getOrderStatusText.ts

```diff
@@ -0,0 +1,16 @@
+import { OrderStatus } from '../types'
+
+export function getOrderStatusText(status: OrderStatus) {
+  switch (status) {
+    case 'open':
+      return 'Active'
+    case 'fulfilled':
+      return 'Fulfilled'
+    case 'cancelled':
+      return 'Cancelled'
+    case 'expired':
+      return 'Expired'
+    default:
```

Comment:
What status strings should convert to pending? May be make sense to do it in this way:
```
case 'pending':
  return 'Pending'
default:
  return 'Unknown'
```